tensorbay/apps/sextant.py

```diff
@@ -22,9 +23,17 @@ class Evaluation:
 
     """
 
-    def __init__(self, evaluation_id: str, benchmark: "Benchmark") -> None:
+    def __init__(
+        self, evaluation_id: str, create_at: int, status: int, benchmark: "Benchmark"
+    ) -> None:
         self.evaluation_id = evaluation_id
         self.benchmark = benchmark
+        self.create_at = create_at
+        self.status = status
+
+    def __repr__(self) -> str:
+        read_time = time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(self.create_at))
```

Comment:
why display time in `repr`?

Reply:
Add time information to help users distinguish evaluations.
I want to add a `title` field to do this later.